modules/security-aai/src/main/java/org/opencastproject/security/aai/ConfigurableLoginHandler.java

```diff
@@ -425,6 +465,14 @@ private String extractEmail(HttpServletRequest request) {
     if (StringUtils.equals(id, bootstrapUserId)) {
       roles.add(new JpaRole(GLOBAL_ADMIN_ROLE, organization));
     }
+	if (headerAffiliation != null) {
+      String affiliation = request.getHeader(headerAffiliation);
+	  List<String> affiliations = Arrays.asList(affiliation.split(";"));
+	  for (String var : affiliations) {
```

Comment:
`var` is such a generic name. Why is this not called `affiliation`?

Reply:
I changed it as you suggested.

Reply:
> 
So, that was why I named it "var". Should I change it back to "var"?

`[ERROR] /home/travis/build/opencast/opencast/modules/security-aai/src/main/java/org/opencastproject/security/aai/ConfigurableLoginHandler.java:[471,23] variable affiliation is already defined in method extractRoles(java.lang.String,javax.servlet.http.HttpServletRequest)`

> I changed it as you suggested.